priv/static/Linux.sh

```diff
@@ -0,0 +1,422 @@
+        echo ""
+        echo "Skipping Docker"
+        echo ""
+    else
+        echo -e "${cyan}${bold}Hang on tight. We will do the optional installation
+        (Chrome, Chromedriver, Docker)"
+        echo ""
+
+        echo -e "${cyan}${bold}1) Chrome
+        (Next step is to add Chrome, Firefox is pre-installed on Ubuntu, but we need Chrome to run automated tests)"
+        echo -e "${white}"
+        sleep 3
+        get "chrome" 
+        echo ""
+
+        echo -e "${cyan}${bold}2) Node.js"
+        echo -e "${white}"
+        sleep 1.5
+        get "nodejs"
+        echo ""
+
+        echo -e "${cyan}${bold}3) chromedriver
+        (Now we are gonna add chromedriver that we also need for tests)"
+        echo -e "${white}"
+        sleep 2
+        get "chromedriver"
+        echo ""
+
+        echo -e "${cyan}${bold}4) Docker"
+        echo -e "${white}"
+        get "docker"
+        echo ""
+    fi
+
+    echo ""
+    echo -e "${cyan}${bold}phx.tools setup is complete!"
+    echo -e "${cyan}${bold}Please restart the terminal and type in the following command:"
+    echo -e "${cyan}${bold}mix phx.new"
+    echo -e "${white}"
+}
+
+phx_tools="
+        ██████╗░██╗░░██╗██╗░░██╗  ████████╗░█████╗░░█████╗░██╗░░░░░░██████╗     
+        ██╔══██╗██║░░██║╚██╗██╔╝  ╚══██╔══╝██╔══██╗██╔══██╗██║░░░░░██╔════╝     
+        ██████╔╝███████║░╚███╔╝░  ░░░██║░░░██║░░██║██║░░██║██║░░░░░╚█████╗░ 
+        ██╔═══╝░██╔══██║░██╔██╗░  ░░░██║░░░██║░░██║██║░░██║██║░░░░░░╚═══██╗ 
+        ██║░░░░░██║░░██║██╔╝╚██╗  ░░░██║░░░╚█████╔╝╚█████╔╝███████╗██████╔╝ 
+        ╚═╝░░░░░╚═╝░░╚═╝╚═╝░░╚═╝  ░░░╚═╝░░░░╚════╝░░╚════╝░╚══════╝╚═════╝░ 
+"
+by="
+                            ██████╗░██╗░░░██╗
+                            ██╔══██╗╚██╗░██╔╝
+                            ██████╦╝░╚████╔╝░
+                            ██╔══██╗░░╚██╔╝░░
+                            ██████╦╝░░░██║░░░
+                            ╚═════╝░░░░╚═╝░░░
+"
+optimum="
+    ░█████╗░██████╗░████████╗██╗███╗░░░███╗██╗░░░██╗███╗░░░███╗██████╗░██╗░░██╗
+    ██╔══██╗██╔══██╗╚══██╔══╝██║████╗░████║██║░░░██║████╗░████║██╔══██╗██║░░██║
+    ██║░░██║██████╔╝░░░██║░░░██║██╔████╔██║██║░░░██║██╔████╔██║██████╦╝███████║
+    ██║░░██║██╔═══╝░░░░██║░░░██║██║╚██╔╝██║██║░░░██║██║╚██╔╝██║██╔══██╗██╔══██║
+    ╚█████╔╝██║░░░░░░░░██║░░░██║██║░╚═╝░██║╚██████╔╝██║░╚═╝░██║██████╦╝██║░░██║
+    ░╚════╝░╚═╝░░░░░░░░╚═╝░░░╚═╝╚═╝░░░░░╚═╝░╚═════╝░╚═╝░░░░░╚═╝╚═════╝░╚═╝░░╚═╝
+"
+
+echo -e "$phx_tools" 
+
+echo -e "$by"
+
+echo -e "$optimum"
+
+sleep 3
+
+echo ""
+
+echo -e "${bblue}${bold}Welcome to our Phx.tools script for linux based OS"
+
+sleep 3
+
+echo ""
+
+echo -e "${bblue}${bold}The following tools and dependencies will be installed if you don't have them
+already installed:"
+
+echo -e "${cyan}${bold}"
+
+echo "1) Curl"
+echo "2) Zsh"
+echo "3) Chrome"
+echo "4) Homebrew"
+echo "5) Chromedriver"
+echo "6) asdf"
+echo "7) Erlang"
+echo "8) Elixir"
+echo "9) Phoenix"
+echo "10) Node.js"
```

Comment:
This is an optional dependency, right?

Reply:
the only optional dep on those line is node.js,
here are the optional deps:
1. Chrome
2. Node.JS
3. Chromedriver
3. Docker

should I remove copy the from the installation message and only show them only in optional installation message

Reply:
Yeah, I prefer not to bog the user with long text. We'll try to explain how it works better on website later on as well as in the blog post.

Reply:
Great, I'll remove it